terraform/services/api-waf/main.tf

```
@@ -52,6 +52,7 @@ module "aws_waf" {
   content_type = "APPLICATION_JSON"
 
   associated_resource_arn = data.aws_lb.api.arn
+  rate_limit              = var.app == "dpc" ? 3000 : 300
```

This is good but seeing as BCDA is the team with the need for strict rate limits it might be better to flip it. So update this as follows and set the default at https://github.com/CMSgov/ab2d-bcda-dpc-platform/blob/a57fd68067485a38805d4e405a94d730fd82abe3/terraform/modules/firewall/variables.tf#L51 to 3000 as well. 

```suggestion
  rate_limit              = var.app == "bcda" ? 300 : 3000
```